utils/webhook/github.handler.ts

```diff
@@ -340,6 +340,41 @@ export async function githubWebhookHandler(
             }
         );
 
+        /** In case if user has added some other labels on issue */
+        const githubLabels = issue.labels.filter(
+            label => label.name !== "linear"
+        );
+
+        const linearCreatedLabels: Array<string> = [];
+        for (const githubLabel of githubLabels) {
+            const existingLinearLabel = await linear.issueLabels({
+                includeArchived: true,
+                filter: {
+                    name: { eqIgnoreCase: githubLabel.name },
+                    team: { id: { eq: linearTeamId } }
+                }
+            });
+
+            /** A label is already created with the same name */
+            if (existingLinearLabel.nodes.length > 0) {
+                linearCreatedLabels.push(existingLinearLabel.nodes[0].id);
+            }
+
+            /** If not, then create it */
+            if (existingLinearLabel.nodes.length === 0) {

```

Comment:
Let's avoid creating labels in Linear since we don't do it (by design) elsewhere